GrafX2.txt

```diff
@@ -8,6 +8,7 @@ URLSite = http://grafx2.chez.com/
 URLDownload = https://gitlab.com/GrafX2/grafX2/-/jobs/3140386057/artifacts/raw/install/grafx2-win32-2.8.3137.win32.exe
 SHA1 = df7ec1ee27dd509c484a277aa33546adc3616817
 SizeBytes = 2198528
+Icon = GrafX2.ico
```

Comment:
If the icon has the same name as the `.txt` file, then you don't need this entry.

Reply:
Done :)  [5bc72ee](https://github.com/reactos/rapps-db/pull/240/commits/5bc72ee8f2742c831528e70fa38ec089223baec6)

Reply:
Okay they are all removed